src/ai/backend/manager/models/base.py

```diff
@@ -417,6 +417,9 @@ def __init__(self, schema: Type[JSONSerializableMixin]) -> None:
         super().__init__()
         self._schema = schema
 
+    def coerce_compared_value(self, op, value):
+        return JSONB()
+
```

Comment:
What is the purpose of this interface

Reply:
I didn't mention that. Here is the full context https://github.com/sqlalchemy/sqlalchemy/discussions/11369
In short, this overridden method allows us to query like this.
```python
vfolder_mount_value = [{"vfid": "VFOLDER_ID"}]
query = sa.select(SessionRow).where(SessionRow.vfolder_mounts.contains(vfolder_mount_value))
```
We can set a query condition on `StructuredJSONObjectListColumn` type column with simple JSON value